### ISSUE TITLE
Add news column

### ISSUE THREAD

**Issue by rajmahendra** [mvc-spec/www.mvc-spec.org#2]
Add a news menu and content to post News and release about the JSR

**Comment by chkal** [comment]
Thanks. Yes, that's on my TODO list. 😀

**Comment by Daniel-Dos** [comment]
Hello friends !
Taking advantage of the discourse, it would be interesting to have a part with other examples of JSR.

**Comment by chkal** [comment]
I agree! I thought about some kind of "cookbook" which lists common patterns and best practices.

I'll enhance the site in the next days and weeks which hopefully will also simplify the process of contributing content to the site. I'll keep you updated.

**Comment by rajmahendra** [comment]
Let me raise another issue for this :)
